blog/2022-04-06-SDL-snowpark/2022-04-06-SDL-snowpark.md

```diff
@@ -12,7 +12,7 @@ hide_table_of_contents: false
 
 This article shows how to create one unified data pipeline that uses Spark to ingest data into Snowflake, and Snowpark to transform data inside Snowflake.
 
-<!--truncate-->
+\<!--truncate-->

```

Comment:
All these escapes to not work for me, see this one for example:
![image](https://github.com/smart-data-lake/smart-data-lake/assets/6121700/41948dc3-d8e7-4249-86a1-cee12ad4dd8a)

Same goes for all escapes further down. Why were they inserted?

Reply:
Thanks for spotting this error! This was a leftover of one of my refactoring attempts.